tests/shared/core/test_domain.py

```diff
@@ -189,7 +189,7 @@ def test_avoid_action_repetition():
     assert len(domain.action_names) == len(domain_with_no_actions.action_names)
```

Comment:
for `test_domain_from_template`, what does this template mean?

Reply:
No idea what the `template` does in there 😂 I think it's `test_domain_from_file`?

Reply:
I'm ... just gonna leave it.